app/models/alaveteli_pro/request_summary.rb

```diff
@@ -76,12 +76,14 @@ def self.create_from(request)
   end
 
   def self.attributes_from_request(request)
+    user = request.user.is_a?(User::AnonymousUser) ? nil : request.user
```

Comment:
Just to add to the commit message, `AlaveteliPro::RequestSummary` has a `belongs_to :user` association, so when we pass a `User::AnonymousUser` that obviously doesn't work because the association setter is expecting an `ActiveRecord` object.

On WhatDoTheyKnow we do have request summaries without an associated user:

```ruby
AlaveteliPro::RequestSummary.where(user_id: nil).count
# => 1002
```